packages/electron-updater/src/NsisUpdater.ts

```diff
@@ -27,10 +27,6 @@ export class NsisUpdater extends BaseUpdater {
       downloadUpdateOptions,
       fileInfo,
       task: async (destinationFile, downloadOptions, packageFile, removeTempDirIfAny) => {
-        if (hasQuotes(destinationFile) || (packageFile != null && hasQuotes(packageFile))) {
-          throw newError(`destinationFile or packageFile contains illegal chars`, "ERR_UPDATER_ILLEGAL_FILE_NAME")
-        }
```

Comment:
I am moving the check to the verifier, since I think that is the more appropriate to do this!